test/albumTest.js

```diff
@@ -197,3 +197,109 @@ describe('/users/:user_id/albums GET', () => {
     });
   });
 });
+
+describe('/users/albums/:id/photos GET', () => {
+  let request;
+  let requestNotAlbumBought;
+  let user;
+  beforeEach(done => {
+    User.create({
+      name: 'Franco',
+      surname: 'Coronel',
+      email: 'franco.coronel@wolox.com.ar',
+      password: 'passwordFC'
+    }).then(newUser => {
+      user = newUser;
+      Album.create({ userId: user.id, albumId: 1, title: 'title test' }).then(() => {
+        Album.create({ userId: user.id, albumId: 2, title: 'title test 2' }).then(() => {
+          request = chai.request(server).get('/users/albums/1/photos');
```

Comment:
Remember to use dynamic ids, yo should add a `album` param to the function  and use `${album.id}`